### PR TITLE
Add ParseDeviceLog and update the README

### PR DESCRIPTION
**Pull request by cawhitworth**
Copy the ParseDeviceLog folder from Azure/azure-sphere-gallery and update the README.

Again, this looks bigger than it is - the only delta is in README; everything else is a straight as-is copy from azure-sphere-gallery.

### CODE REVIEW

File: ParseDeviceLog/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+         `c:>./parse-devicelog.py -d 22.07 -f <path to device log file>/AzureSphere_DeviceLog_113.bin`
+
+      The AzureSphere_DeviceLog_113.bin file's contents will be displayed in the CLI.
+
+      ![alt text](./ParseDeviceLog.png)
+
+### Expected support for the code
+
+This code is not formally maintained, but we will make a best effort to respond to and address any issues you encounter.
```
Comment (jamesadevine) [inline ParseDeviceLog/README.md:65]: With this move, is the code now formally maintained?

Comment (cawhitworth) [reply ParseDeviceLog/README.md:65]: I believe that was the intention - @mikehall-ms can correct me if I'm wrong, but we (MS) have to maintain this as the log format is tied to our OS builds.

Comment (cawhitworth) [reply ParseDeviceLog/README.md:65]: (and so this README should be updated)

Comment (cawhitworth) [reply ParseDeviceLog/README.md]: Removed this section

**Review by jamesadevine — APPROVED** [review]

**Review by mikehall-ms — APPROVED** [review]

**Pull request merged**